contents/uix/uix.md

```diff
@@ -45,3 +47,65 @@ Just like Software Development, UX Design is an iterative process. It involves m
+
+## The Process of UX Design
+
+As stated above, UX is a highly iterative process to find the right balance for the target audience. Many times the functionality of a product is prioritised over the experience it gives, however this should not be the case when experience is the deciding factor in delighting or frustrating users.
+
+Catering the right experiences for users does not have to be difficult. The stages of UX Design are logical and can be decomposed into four main stages as shown.
+1. Preliminary Research
+1. Prototyping
+1. User Testing
+1. Maintenance
+
+### Preliminary Research
+The first stage is Preliminary Research. This is where you would:
+  * Understand your target audience through [Contextual Inquiry](http://www.usabilitybok.org/contextual-inquiry), which involves interviewing users to find out about their wants, needs and pain-points, in the setting of them using the actual product and/or products from competitors
+  * Gather the tools needed for the project such as wireframing or prototyping [tools](https://medium.com/@Mockplus/the-10-best-wireframing-and-prototyping-tools-for-designers-a808e81ecadf) for creating low to high fidelity products for testing
+  * Depending on whether the project is hypothetical (taking a known product and re-designing it) or client-based (working for businesses or non-profits), discuss and create a project plan with stakeholders
+
+```Deliverables```
+  * [Personas](https://static1.squarespace.com/static/519deddfe4b07b846eef9842/55915575e4b0a919cf5a0da5/55a5b693e4b0576c4f936a78/1436923548676/11_all_with_eps_brands.jpg?format=750w) - User profiles that are representative of the target audiences
+  * [User Stories](https://i.pinimg.com/originals/00/ac/37/00ac379ca5b9c28de3d56bdea4580e2e.jpg) - Representation of the most important user actions or motivations for each persona
+  * [User Scenarios](http://www.davedoyle.com/prof/portfolio/images/large/petcare_scenario.gif) - Case studies of how users use the products, supplementing User Stories to allow you to better empathize with the users
+  * [User Flows](https://cdn-images-1.medium.com/max/1600/0*aO_6rai_fTnSe5rh.) - Depiction of how users navigate through the product for each scenario
+  
+### Prototyping
+After Preliminary Research, the next stage is Prototyping. This is where you would:
+  * [Test](https://usabilitygeek.com/usability-testing-prototypes/) whether the product flow you came up with is smooth
+  * Create [prototypes](https://www.mockplus.com/blog/post/wireframe-mockup-prototype-selection-of-prototyping-tools), or Minimum Viable Product (MVP), for further testing and refinement
+  * Get a clear sense of how users would interact with the final product
```

Comment:
How can we "get a clear sense of how users would interact with the final product"? If possible, clarify how this particular point differs from the user flows in the previous stage and the prototypes to be used for user testing in the next stage.

(On a broader note, you could consider integrating the deliverables with the main points so we can see how each point links to a particular deliverable. There's probably no one-to-one correspondence, but something to think about?)